crates/wasmtime/src/frame_info.rs

```diff
@@ -41,7 +41,7 @@ pub struct GlobalFrameInfoRegistration {
 struct ModuleFrameInfo {
     start: usize,
     functions: BTreeMap<usize, FunctionInfo>,
-    module: Arc<Module>,
+    module: Arc<CompiledModule>,
```

Comment:
How come this changed to `CompiledModule`? That's retaining a lot more information here than `Module` previously was

Reply:
Returned back to store only Module, and added ModuleCode (which is only CodeMemory + GdbJitRegistration)